resources/tests/test_api.py

```diff
@@ -30,21 +30,21 @@ def test_1_create_index(self):
 
     def test_2_video_docs(self):
         doc_list = api.get_video_docs()
-        self.assertTrue(doc_list is not None and len(doc_list) > 10)
+        self.assertEqual(len(doc_list), 12)
 
     def test_3_video_items(self):
         video_items = api.get_video_items()
-        self.assertTrue(len(video_items.get('video_list', '')) > 15)
+        self.assertEqual(len(video_items.get('video_list', '')), 15)
 
     def test_4_video_sec(self):
         menu_url = 'http://latelelibre.fr/emissions/thibault-o-tour-du-monde/'
         video_list = api.get_video_sec(menu_url)
-        self.assertTrue(video_list is not None and len(video_list) > 8)
+        self.assertEqual(len(video_list), 12)
 
     def test_5_video_hackaround(self):
         menu_url = 'http://latelelibre.fr/series/une-bien-belge-histoire/'
         video_list = api.get_video_sec(menu_url)
-        self.assertTrue(video_list is not None and len(video_list) > 4)
+        self.assertEqual(len(video_list), 5)
```

Comment:
Please, leave it as is. This condition is website content dependent and will vary with the time.